islandora.module

```diff
@@ -624,6 +631,10 @@ function islandora_permission() {
       'title' => t('Replace datastreams'),
       'description' => t('Add new datastream content as latest version.'),
     ),
+    'ISLANDORA_MANAGE_ORPHANED_OBJECTS' => array(
```

Comment:
This is a literal string note the quotes. I think it is in the process of becoming a constant.  The constant will need to be declared. This is usually done at the top of the module file.

Reply:
Is that with the "define" function in the Permissions section? I think I got that in the latest commit.

Reply:
@bondjimbond now that you have `define`d the constant you can call it without quotes. 

If you can `ISLANDORA_MANAGED_ORPHANED_OBJECTS` php will see `'view and delete a list of orphaned objects'`

If you can `'ISLANDORA_MANAGED_ORPHANED_OBJECTS'` php will see `'ISLANDORA_MANAGED_ORPHANED_OBJECTS'`